File: lib_common/src/main/java/com/fly/tour/common/provider/IMeProvider.java
package com.fly.tour.common.provider;

import android.support.v4.app.Fragment;

import com.alibaba.android.arouter.facade.template.IProvider;

/**
 * Description: <IMeProvider><br>
 * Author:      mxdl<br>
 * Date:        2019/5/23<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
public interface IMeProvider extends IProvider {
    Fragment getMainMeFragment();
}

File: lib_common/src/main/java/com/fly/tour/common/provider/IMeProvider.kt
package com.fly.tour.common.provider

import androidx.fragment.app.Fragment

import com.alibaba.android.arouter.facade.template.IProvider

/**
 * Description: <IMeProvider><br>
 * Author:      mxdl<br>
 * Date:        2019/5/23<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
</IMeProvider> */
interface IMeProvider : IProvider {
    val mainMeFragment: Fragment
}
